Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM registry.hub.docker.com/pandoc/latex:2.14.2-ubuntu
+
+RUN mkdir /source
+COPY . /source/.
+
+RUN apt-get update &&\
+    apt-get install -y \
+        curl \
+        git \
+        fonts-league-spartan \
```

Comment:
League Spartan has some stylistic differences with the original Spartan (which is no longer maintained), there is however also [Spartan MB](https://github.com/MattBaileyDesign/Spartan-MB) from the original designer. This still requires some minor tweaks to TechCheck.cls in order to fix sizing issues

Reply:
I started digging into the fonts and yeah ... scaling is an issue.

I'll keep digging on that one. I just need to sink-in the time to make it look correct.